src/components/common/DocumentModal/DocumentModal.tsx

```diff
@@ -11,8 +11,9 @@ interface DocumentModalProps {
   filename: string
   url: string
   open: boolean
-  setOpen: (open: boolean) => void
+  onClose: () => void
```

Comment:
The modal only needs an "onClose" method, as opening is done by changing the state of `open` in the parent component.